### DISCUSSION TITLE
Symfony 6.1 to 6.0, any help ?

### DISCUSSION THREAD

**Discussion by EzHeisenberg** [symfony/symfony#46898] [Q&A]
Hello everyone, 
Today I'm in a case where my host doesn't support php 8.1... this is annoying for me because I built a Symfony application in version 6.1 => php >8.1
I would like to have some hints, because I already tried to change my Symfony and php version in my composer.json (having deleted my composer.lock) but it generated several errors like this: 

```zsh
php bin/console cache:clear
Symfony\Component\ErrorHandler\Error\ClassNotFoundError^ {#90
  #message: """
    Attempted to load class "DebugBundle" from namespace "Symfony\Bundle\DebugBundle".\n
    Did you forget a "use" statement for another namespace?
    """
  #code: 0
  #file: "./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php"
  #line: 135
  trace: {
    ./vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:135 { …}
    ./vendor/symfony/http-kernel/Kernel.php:382 { …}
    ./vendor/symfony/http-kernel/Kernel.php:766 { …}
    ./vendor/symfony/http-kernel/Kernel.php:128 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:166 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:72 { …}
    ./vendor/symfony/console/Application.php:171 { …}
    ./vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php:54 { …}
    ./vendor/autoload_runtime.php:29 { …}
    ./bin/console:11 {
      › 
      › require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
      › 
      arguments: {
        "<le_path_de_mon_app>/vendor/autoload_runtime.php"
      }
    }
  }
}
2022-07-11T01:31:31+00:00 [critical] Uncaught Error: Class "Symfony\Bundle\DebugBundle\DebugBundle" not found

```
Having several such errors and trying to correct them, I defer to you. I am not trying to solve the problem with the above code, but to get a clue about this downgrade.

**Comment by ahundiak** [comment]
Your best bet is to change hosts though I suspect that may be impractical.

Try completely deleting your var directory.  Lots of cached code under there which the cache clear command initially tries to use.  Probably won't help.

You may end up having to create a new 6.0 project and then basically copy/pasting your 6.1 code over.  Bit of a pain perhaps but it should get the job done.

**Comment by RSickenberg** [comment]
Have you set `"symfony/debug-bundle": "6.0.*"` to your composer.json (require-dev) and also in `bundles.php` with that line: `Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true]` ?

**Reply by zodman** [reply]
I have the same error and its like you said.... debug-bundle in require-dev and dev=>true in bundles.php

**Reply by TossouSeomaniak** [reply]
I did it but I still have the same error during production

**Comment by kdosiodjinud** [comment]
Hi, for me was solution set APP_ENV env variable before run `composer install --no-dev`

for example `export APP_ENV=prod && composer install --no-dev`
